Azure-Samples/modern-data-warehouse-dataops
# [E2E Databricks]: feat: improving deployment experience and functionality

# Type of PR
- Documentation changes
- Code changes
- Test changes
- CI-CD changes

## Purpose
- To improve the overall functionality and experience with the sample - mostly summarized on the item #888 

## Does this introduce a breaking change? If yes, details on what can break
Yes, all the changes were already tested before merging the feature branch, however the changes can break the following:
- Devcontainer
- Bicep deployment
- CICD 

## Author pre-publish checklist

## Validation steps
- Open sample on the devcontainer
- Run deployment and check that there are no errors
- Run ADF pipeline and CICD pipelines with no errors

## Issues Closed or Referenced

- Closes  
- #888 
- #831 
- #886 
- #879 
- #898 
- #881 
- #897 
- #918 
- #903 
- #940 
- #929
